packages/rocketchat-importer-csv/server.js

```diff
@@ -254,7 +298,7 @@ Importer.CSV = class ImporterCSV extends Importer.Base {
 							const creator = this.getUserFromUsername(msg.username);
 							if (creator) {
 								const msgObj = {
-									_id: `csv-${ csvChannel.id }-${ msg.ts }`,
+									_id: `csv-${ csvChannel.id }-${ roomCounter }-${ msg.ts }`,
```

Comment:
Only issue with this is that people who try to reimport data that has been imported before this change will see their data again...